.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -412,7 +412,10 @@ jobs:
 
     - name: Install Catalyst
       run: |
-        python${{ matrix.python_version }} -m pip install $GITHUB_WORKSPACE/dist/*.whl --extra-index-url https://test.pypi.org/simple
+        python${{ matrix.python_version }} -m pip install --no-deps $GITHUB_WORKSPACE/dist/*.whl
+        pip install --upgrade pennylane-lightning==0.36.0
+        pip install --upgrade git+https://github.com/PennyLaneAI/pennylane@v0.36.0-rc0
```

Comment:
I'm assuming this is a temporary fix just for testing the RC? Always good to add a comment in that case.

> pip install --upgrade pennylane-lightning==0.36.0

Does that mean lightning 0.36 is already available?

Reply:
Problem with no `--no-deps` is doesn't install jax either. I think the solution is that you install pennylane and lightning first, then the requirements of Catalyst should be satisfied right (assuming `v0.36.0-rc0` satisfies `>=0.36` 🤔 ).

Reply:
Alternatively publishing a test release for all three packages to test pypi _without_ the minimum version bump would then allow us to test properly once we do bump the dependency versions :)

Reply:
Yes, this was just an initial test. Thanks for the review :)

Reply:
https://github.com/PennyLaneAI/catalyst/pull/715/commits/6893f8ba98124852ad98bcacfd42d07f0c8b0bc8

Reply:
Updated all of them to use the rc.